README.md

```diff
@@ -48,11 +48,12 @@ npm run stylelint
 ## Prettier
 
 This config also includes a prettier config which can be extended to format `.scss`.
+Using the `stylelint-prettier` plugin, prettier changes are exposed as stylelint rule violations.
```

Comment:
Let's link `stylelint-prettier` back to the github repo for folks who want to dig a bit deeper into this.